### DISCUSSION TITLE
"Current Charge View" lacks current cost

### DISCUSSION THREAD

**Discussion by nunojusto** [jheredianet/Teslamate-CustomGrafanaDashboards#60] [Ideas]
Hello. Thanks for all the hard work and amazing output.
But i think "Current Charge View" needs the cost count when it's a charge on a known (geo fence) charger that has a cost already set.

**Comment by jheredianet** [comment]
Hi, first of all, thank you for your feedback. I'm moving this to discussion, since it is not an issue. 

Your idea is good, but we need to have in mind that Grafana need to read values from the database in a timeline, that's why there is other "browse charges" dashboard where you can explore the price when the charge session is done (with acumulative values). 
There are other cases where people have different prices in the same Geofence depending in the hour of day or weekend, so it will need some script (or programing) that we cannot include in Grafana, since it is not a programming language, just a way to represent the data from de Postgres database. 
I'll try to analyze further. 

Cheers

**Reply by stauffenberg2020** [reply]
Can we somehow leverage the capabilities of [TeslaMateAgile](https://github.com/MattJeanes/TeslaMateAgile)

**Reply by jheredianet** [reply]
Yes, but it's most a work on the backend (TeslaMate), not on a Grafana dashboard.
